tuf/client/client.go

```diff
@@ -402,7 +402,7 @@ func (c *Client) downloadTargets(role string) error {
 			return err
 		}
 		if c.local.Snapshot == nil {
-			return ErrMissingMeta{role: role}
+			return data.ErrMissingMeta{Role: role}
```

Comment:
:|  it should probably be some kind state error, like that snapshot should have been loaded by now, but yes `Role: "snapshot"` would definitely be better than now, thanks :)